docs/_includes/api-options.adoc

```diff
@@ -0,0 +1,149 @@
+//= Ruby API Options
+
+// ref issue 562.
+
+.Ruby API options
+[cols="<15m,<35,<20,<20,<25"]
+|====
+|Name |Description |Default value |Allowed values |CLI equivalent
+
+|attributes
+|Passes map of document attributes.
+|_Empty_
+|Select from <<builtin-attributes>>, or user-defined.
+|`-a`, `--attributes`
+
+|backend
+|Output file format
+|html5
+|html5, docbook5, docbook45, manpage
+|`-b`, `--backend`
+
+|base_dir
+|Base directory containing the document and resources.
+|The directory containing the source file, or the working directory if the source is read from a stream.
+|_path_
+|`-B`, `--base-dir`
+
+|catalog_assets
+|?
+|
+|
+|_none_
+
+|compact
+|Not used
+|false
+|true or false
+|_none_
+
+|converter
+|?
+|
+|
+|_none_
+
+|doctype
+|Document type
+|article
+|article, book, manpage, inline
+|`-d`, `--doctype`
+
+|eruby
+|Specifies the eRuby implementation to use for executing the custom converter templates written in ERB.
+|erb
+|erb, erubis
+|`-e`, `--eruby`
+
+|extensions
+|List of file extensions for asciidoc source files.
```

Comment:
Nope. The extensions options is a Ruby block that registers (and possibly defines) Asciidoctor extensions for this instance of the processor.

The default value is nil. The allowed value is a Ruby block that conforms to the Asciidoctor extensions API.